ledger/accountdb_test.go

```diff
@@ -1269,3 +1269,30 @@ func TestResourcesDataAsset(t *testing.T) {
 	a.Equal(assetParamsEmpty, rd.GetAssetParams())
 	a.Equal(assetHoldingEmpty, rd.GetAssetHolding())
 }
+
+func TestBaseAccountDataIsEmpty(t *testing.T) {
```

Comment:
I mean at least less empty cycles spent on iterations

Reply:
I have added structure testing : I'm comparing the json marshaling of the structure against known good. This would ensure that not only we have 17 fields, but the fields are what we expect them to be, and that there are no other fields added.